.golangci.yml

```diff
@@ -96,6 +97,28 @@ linters-settings:
       mnd:
         # don't include the "operation" and "assign" checks
         checks: argument,case,condition,return
+  linters-settings:
+  gci:
+    # DEPRECATED: use `sections` and `prefix(github.com/org/project)` instead.
+    # local-prefixes: github.com/rilldata/rill-developer
+    # Section configuration to compare against.
+    # Section names are case-insensitive and may contain parameters in ().
+    # The default order of sections is `standard > default > custom > blank > dot`,
+    # If `custom-order` is `true`, it follows the order of `sections` option.
+    # Default: ["standard", "default"]
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
+      - prefix(github.com/rilldata/rill-developer) # Custom section: groups all imports with the specified Prefix.
```

Comment:
Remove the prefix match here. It doesn't work anyway since we use `rilldata/rill`, not `rilldata/rill-developer`